cast/src/lib.rs

```diff
@@ -296,35 +296,44 @@ where
                 .provider
                 .get_block_with_txs(block)
                 .await?
-                .ok_or_else(|| eyre::eyre!("block {:?} not found", block))?;
+                .ok_or_else(|| eyre::eyre!("block {:?} not found", block))
+                .unwrap();
```

Comment:
Can we make this a question mark again instead of unwrapping?

Reply:
Thank you, addressed!